staticd/static_vty.c

```diff
@@ -325,7 +325,13 @@ static int static_route_leak(struct vty *vty, const char *svrf,
 
 		dnode = yang_dnode_get(vty->candidate_config->dnode, ab_xpath);
 		if (!dnode) {
-			/* Silently return */
+			if (vty)
```

Comment:
`vty` is always passed to the function, no need for this check.

Reply:
fix'd